### DISCUSSION TITLE
Welcome to Smathchat Discussions!

### DISCUSSION THREAD

**Discussion by zoelesv** [zoelesv/Smathchat#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Tayyab-R** [comment]
Hi @zoelesv,
I hope you are doing well. I am Muhammad Tayyab from Pakistan and I am looking forward to learn and grow from this community.
I hope future holds good fortune for this community.

**Reply by zoelesv** [reply]
Hi @Tayyab-R ,
Hope you’re doing well too. It's nice to see a supportive community here. 
Looking forwards to share more ideas and collaborate. 👍
